src/accounting.rs

```diff
@@ -2,119 +2,168 @@ use cketh_common::eth_rpc_client::providers::RpcApi;
 
 use crate::*;
 
-pub fn get_request_cost(
+/// Returns the cycles cost of a JSON-RPC request.
+pub fn get_json_rpc_cost(
     source: &ResolvedJsonRpcSource,
-    json_rpc_payload: &str,
+    payload_size_bytes: usize,
```

Comment:
Why is `payload_size_bytes` of type `usize` but  `max_response_bytes` is `u64`?

Reply:
Switching `payload_size_bytes` to `u64` causes a lot of unnecessary conversions (since string/vec lengths are always returned as `usize`), whereas `max_response_bytes` is always a `u64` due to being passed by the developer. It's a bit of an awkward situation, and it bugs me as well. 

I'll explore changing this in a separate PR for a cleaner diff.